client-cli/src/command.rs

```diff
@@ -111,6 +111,27 @@ pub enum Command {
         )]
         disable_fast_forward: bool,
     },
+    #[structopt(
+        name = "download_tx",
+        about = "Download a plain transaction by a given transaction id"
+    )]
+    DownloadTx {
+        #[structopt(name = "name", short, long, help = "Name of wallet")]
+        name: String,
+        #[structopt(name = "tx_id", short, long, help = "transaction id")]
+        tx_id: String,
+    },
+
+    #[structopt(
+        name = "sync_tx",
+        about = "Synchronize a plain transaction that view key not included"
```

Comment:
in `about`, it may good to list the transaction types this applies to

Reply:
also how about
./client-cli transaction download --name a --type staking
sub-command of transaction

download_tx is too long to type

Reply:
yes, I think so.